### PR TITLE
Created new POM class 

### PR DESCRIPTION
**Pull request by ghost**
Please review

### CODE REVIEW

File: ComcastCRMGUIFramework/src/main/java/com/comcast/crm/objectrepositoryutility/CreateProductPage.java [hunk 1]
```diff
@@ -0,0 +1,9 @@
+package com.comcast.crm.objectrepositoryutility;
+
+public class CreateProductPage {
+	
+	//code1
+	//code
```
Comment (qspidersseleniumoar) [inline ComcastCRMGUIFramework/src/main/java/com/comcast/crm/objectrepositoryutility/CreateProductPage.java:6]: change the xpath
do not hardcode the date

**Review by qspidersseleniumoar — COMMENTED** [review]
added few comments

File: ComcastCRMGUIFramework/src/main/java/com/comcast/crm/objectrepositoryutility/CreateProductPage.java (hunk 1 above)
Comment (TestyantraSolutions) [reply ComcastCRMGUIFramework/src/main/java/com/comcast/crm/objectrepositoryutility/CreateProductPage.java:6]: fixed

**Comment by qspidersseleniumoar** [comment]
merged

**Pull request merged**